code/src/terrat_vcs_service_github/terrat_vcs_service_github_assets.ml

```diff
@@ -0,0 +1,198 @@
+
+  let repo_config_err_access_control_unlock_match_parse_err =
+    read "repo_config_err_access_control_unlock_match_parse_err.tmpl"
+
+  let repo_config_err_apply_requirements_approved_all_of_match_parse_err =
+    read "repo_config_err_apply_requirements_approved_all_of_match_parse_err.tmpl"
+
+  let repo_config_err_apply_requirements_approved_any_of_match_parse_err =
+    read "repo_config_err_apply_requirements_approved_any_of_match_parse_err.tmpl"
+
+  let repo_config_err_apply_requirements_check_tag_query_err =
+    read "repo_config_err_apply_requirements_check_tag_query_err.tmpl"
+
+  let repo_config_err_depends_on_err = read "repo_config_err_depends_on_err.tmpl"
+  let repo_config_err_drift_schedule_err = read "repo_config_err_drift_schedule_err.tmpl"
+  let repo_config_err_drift_tag_query_err = read "repo_config_err_drift_tag_query_err.tmpl"
+  let repo_config_err_glob_parse_err = read "repo_config_err_glob_parse_err.tmpl"
+
+  let repo_config_err_hooks_unknown_run_on_err =
+    read "repo_config_err_hooks_unknown_run_on_err.tmpl"
+
+  let repo_config_err_hooks_unknown_visible_on_err =
+    read "repo_config_err_hooks_unknown_visible_on_err.tmpl"
+
+  let repo_config_err_pattern_parse_err = read "repo_config_err_pattern_parse_err.tmpl"
+  let repo_config_err_unknown_lock_policy_err = read "repo_config_err_unknown_lock_policy_err.tmpl"
+
+  let repo_config_err_window_parse_timezone_err =
+    read "repo_config_err_window_parse_timezone_err.tmpl"
+
+  let repo_config_err_workflows_apply_unknown_run_on_err =
+    read "repo_config_err_workflows_apply_unknown_run_on_err.tmpl"
+
+  let repo_config_err_workflows_apply_unknown_visible_on_err =
+    read "repo_config_err_workflows_apply_unknown_visible_on_err.tmpl"
+
+  let repo_config_err_workflows_plan_unknown_run_on_err =
+    read "repo_config_err_workflows_plan_unknown_run_on_err.tmpl"
+
+  let repo_config_err_workflows_plan_unknown_visible_on_err =
+    read "repo_config_err_workflows_plan_unknown_visible_on_err.tmpl"
+
+  let repo_config_err_workflows_tag_query_parse_err =
+    read "repo_config_err_workflows_tag_query_parse_err.tmpl"
+
+  let plan_complete = read "plan_complete.tmpl"
+  let apply_complete = read "apply_complete.tmpl"
+  let plan_complete2 = read "plan_complete2.tmpl"
+  let apply_complete2 = read "apply_complete2.tmpl"
+  let automerge_failure = read "automerge_error.tmpl"
+  let premium_feature_err_access_control = read "premium_feature_err_access_control.tmpl"
+
+  let premium_feature_err_multiple_drift_schedules =
+    read "premium_feature_err_multiple_drift_schedules.tmpl"
+
+  let premium_feature_err_gatekeeping = read "premium_feature_err_gatekeeping.tmpl"
+  let repo_config_merge_err = read "repo_config_merge_err.tmpl"
+  let gate_check_failure = read "gate_check_failure.tmpl"
+  let tier_check = read "tier_check.tmpl"
+  let build_tree_failure = read "build_tree_failure.tmpl"
+end
+
+module Ui = struct
```

Comment:
Did you mean to move this over too?

Reply:
@orbitz Actually, I just noticed I need `Ui` in at least two other files, I know this will make compilation slower (for now), but I'm willing to pay that price temporaly.

Reply:
I gotcha, I think it's OK.  We're going to be re-jiggering this in the (geologically) near future.